### PR TITLE
feat: Allow more results for premium users

### PR DESCRIPTION
**Pull request by fehmer**

### CODE REVIEW

File: frontend/src/ts/db.ts [hunk 1]
```diff
@@ -225,51 +228,74 @@ export async function initSnapshot(): Promise<
...
-      if (result.afkDuration === undefined) result.afkDuration = 0;
-      if (result.tags === undefined) result.tags = [];
+  if (dbSnapshot.results !== undefined) {
+    //merge
+    const oldestTimestamp =
+      dbSnapshot.results[dbSnapshot.results.length - 1].timestamp;
+    const resultsWithoutDuplicates = results.filter(
+      (it) => it.timestamp < oldestTimestamp
+    );
+    console.log("+++++ mergeResults", {
```
Comment (fehmer) [inline frontend/src/ts/db.ts:287]: remember to remove

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/6825795452

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/6826075074

File: frontend/src/ts/pages/account.ts [hunk 2]
```diff
@@ -1047,18 +1048,22 @@ async function fillContent(): Promise<void> {
...
   if (results === false && !ConnectionState.get()) {
     Notifications.add("Could not get results - you are offline", -1, {
       duration: 5,
     });
     return;
   }
+
   TodayTracker.addAllFromToday();
   if (results) {
+    //TODO remove after testing
```
Comment (fehmer) [inline frontend/src/ts/pages/account.ts:1062]: remember to remove

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/6850688415

File: frontend/src/styles/account.scss [hunk 3]
```diff
@@ -377,3 +445,13 @@
   user-select: none;
   background-color: var(--sub-alt-color);
 }
+
+.progress {
```
Comment (fehmer) [inline frontend/src/styles/account.scss:449]: obsolete?

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/6932721939

**Pull request merged**